pyega3/libs/data_client.py

```diff
@@ -70,7 +70,10 @@ def get_stream(self, path, extra_headers=None):
             headers.update(extra_headers)
 
         url = f'{self.url}{path}'
-        with self.session.get(url, headers=headers, stream=True) as r:
+        request_timeout_in_sec = 1800  # 30 minutes
```

Comment:
should we lessen this a bit based on the stress testing result? I think this depends on the internet connection speed as well